picard/util/pipe.py

```diff
@@ -37,50 +37,44 @@
 
 
 class PipeError(Exception):
-    pass
+    MESSAGE = None
+
+    def __init__(self, *messages):
+        if self.MESSAGE:
+            self.messages = (self.MESSAGE, ) + tuple(messages)
+        else:
+            self.messages = tuple(messages)
+
+    def __str__(self):
+        messages_str = "\n  ".join(str(m) for m in self.messages)
+        if not messages_str:
+            messages_str = "unknown"
+        return f"ERROR: {messages_str}"
 
 
 class PipeErrorInvalidArgs(PipeError):
     MESSAGE = "ERROR: Pipe() args argument has to be iterable"
```

Comment:
With new code, "ERROR: " part isn't needed anymore (it is added in `PipeError.__str__()`